web-client/src/presenter/sequences/submitAddPractitionerSequence.ts

```diff
@@ -48,6 +48,7 @@ export const submitAddPractitionerSequence = [
           {
             emailAvailable: afterSuccess,
             emailInUse: [
+              setScrollToErrorNotificationAction,
```

Comment:
Otherwise, if an existing email alert happened, the user just saw the just-clicked-but-apparently-unsuccessful submit button without any scrolling to the error.